docker-compose.yml

```diff
@@ -60,11 +47,16 @@ services:
   nginx:
     profiles:
       - backend
-    image: nginx:1.27.0
+    build:
+      context: ./nginx
     ports:
       - "8063:8063"
     links:
       - web
+    environment:
+      PORT: 8063
+      NGINX_WORKERS: 1
+      NGINX_UWSGI_PASS: "web:8011"
     volumes:
       - ./config/nginx.conf:/etc/nginx/conf.d/web.conf
```

Comment:
With the changes here, this should be changed to:

```yaml
volumes:
  - ./config:/etc/nginx/templates
  - ./frontends/mit-open/build:/src/frontends/mit-open/build
```

Although, I think we can also remove that volume mount for the frontend build now right?